### ISSUE TITLE
PSF tails in NIR camera at wavelength > 9000A

### ISSUE THREAD

**Issue by julienguy** [desihub/desispec#1126]
The amplitude of the Lorentzian tails of the spectrograph PSF is small for wavelength<9000A but increases rapidly at larger wavelength.

This has to be accounted for in the pipeline because it results in fiber cross-talk and an excess of flux at wavelength >9000A for faint fibers next to a bright one.

This is visible in exposure 68790 where a very bright star was on fiber 4504 (SP9).
The following figures show the cross-dispersion profiles around this fiber averaged in bins of 50 rows, and the fraction of light 7 pixel away from the trace center as a function of wavelength.
![scattered-light-profile-z9](https://user-images.githubusercontent.com/5192160/106826554-000f3680-663c-11eb-878a-1767067d0722.png)
![scattered-light-profile-r9](https://user-images.githubusercontent.com/5192160/106826563-030a2700-663c-11eb-9b09-d6a0682c5b5d.png)
![scattered-light-profile-b9](https://user-images.githubusercontent.com/5192160/106826567-04d3ea80-663c-11eb-8179-f3ac7a6a1671.png)

 
![scattered-light-vs-wavelength-z9](https://user-images.githubusercontent.com/5192160/106826527-eff75700-663b-11eb-8281-df7cf1f63ab7.png)
![scattered-light-vs-wavelength-r9](https://user-images.githubusercontent.com/5192160/106826572-09000800-663c-11eb-8881-eafb52bbd619.png)
![scattered-light-vs-wavelength-b9](https://user-images.githubusercontent.com/5192160/106826575-0a313500-663c-11eb-87a7-b9a3b7020fac.png)

This effect is likely the cause of the reported excess of flux at the red end of some spectra.
For instance:
targetid=39633345017022486  sp=4  fiber=2123 tileid=80613

![Figure_1](https://user-images.githubusercontent.com/5192160/106827315-7bbdb300-663d-11eb-953d-e5a73aa9f3b3.png)
which fiber trace in the CCD is next to a bright star.
```
plot_frame -i /global/cfs/cdirs/desi/spectro/redux/blanc/exposures/20201223/00069605/cframe-*4-00069605.fits --fiber 2123,2124 --rebin 8 --legend &
```
![Figure_1b](https://user-images.githubusercontent.com/5192160/106827748-db1bc300-663d-11eb-8583-ffee66575d3f.png)
![Figure_1c](https://user-images.githubusercontent.com/5192160/106827754-dce58680-663d-11eb-9115-badae4815464.png)

**Comment by julienguy** [comment]
Other fibers with this excess of flux at wavelength >9000A

TARGETID PETAL_LOC  FIBER TILEID BRIGHT_FIBER
---------------------------------------------------
39633345017022486 4 2123 80613 2124
39633341695136311 7 3693 80613 3694
39633345012827975 4 2452 80613 2451
39633348351493497 6 3022 80613 3023

**Comment by astro-blue** [comment]
Adding a few more examples from BGS and ELGs. Figures can be found at https://github.com/astro-blue/DESI_VI/blob/master/VI_merging_Blanc/Examples_of_red_calibration.ipynb
---BGS---
Targetid, fiberid, tileid
39633331532336363 4216 80613
39633331536529636 4303 80613
39633351635635740 2680 80613
39633328067840127 1085 80613
39633351648216655 3024 80613
39633358170361009 2674 80613
39633331519754738 423 80613
39633331523946719 432 80613
39633331528139161 4855 80613
39633334942305917 4304 80613
39633334946496849 4190 80613
39633334946497323 4187 80613
39633331515556176 1407 80613
39633341682551782 3319 80613
39633324657871646 4701 80613
39633328109783281 4188 80613
39633338306138124 1509 80613
39633338310331015 1512 80613
39633324620124367 967 80613
39633324624315881 989 80613
39633328105588744 4134 80613
39633328097198812 4854 80613
39633328097199882 4565 80613
39633351656607731 3021 80613
39633334925529100 1510 80613
39633341699326713 3695 80613
39633338318718273 3433 80613
39633338327107240 3701 80613
39633341690937412 3703 80613
39633334917140869 1261 80613
39633345000246228 1578 80613
39633345000246379 1551 80613
39633345004442231 1731 80613
39633338331304315 3697 80613
39633338335497905 3836 80613

---ELG---
targetid,fiberid,tile
39633324649480703 4991 80608
39633334912943844 1149 80608
39633328097200958 4561 80608
39633338301940923 1818 80608
39633328080423088 773 80608
39633354902998497 2445 80608

**Comment by julienguy** [comment]
Issue addressed in PR #1138 .

**Comment by sbailey** [comment]
fixed by PR #1138; closing.
